lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +28,16 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(text, str):
+        punctuation = '''!()-[]{};:'"\,<>./?@#$%^&*_~'''
+        res = " "
```

Comment:
By setting `res` to be `" "` you are adding a space to the sequence of preprocessed text, consider initializing it as simple as `""`